hail/python/hail/backend/service_backend.py

```diff
@@ -203,7 +203,7 @@ async def create(*,
                      worker_cores: Optional[Union[int, str]] = None,
                      worker_memory: Optional[str] = None,
                      name_prefix: Optional[str] = None,
-                     token: Optional[str] = None,
+                     credentials_token: Optional[str] = None,
```

Comment:
ServiceBackend.create is a non-public API and the word token is overloaded in this class. Now `token` only means the idempotency token of a batch.